node/core/approval-voting/src/tests.rs

```diff
@@ -740,6 +740,22 @@ impl ChainBuilder {
 	}
 }
 
+fn session_info(keys: &[Sr25519Keyring]) -> SessionInfo {
```

Comment:
We should probably prefix this with `dummy_` or `make_` to make it explicit this is only every used for test and not some internal `fn`.

Reply:
Agree, although this is a private function in tests module :P

Reply:
Should have Prefix It With "Micro it:" 🌻